.deployment/docker-compose-overrides.yml

```diff
@@ -2,10 +2,10 @@ services:
   server:
     entrypoint: /.deployment/dev-entrypoint.sh
     environment:
+      - "NODE_ENV=production"
```

Comment:
I am guess this is appended so it overrides the earlier `NODE_ENV` from the other docker-compose file. Is that right?

Reply:
Yeah. But it might not be necessary, the only reason I added it was to make quick start easier (without user accounts) with adding `NODE_ENV=default` in the `docker/docker-compose.yml`. It might not be needed if we use `docker-compose run`.

```sh
 docker-compose --file docker-compose.yml run -p 8888:8888 -p 8889:8889 -e "NODE_ENV=default" server
```